lib/createChangelog.js

```diff
@@ -1,23 +1,45 @@
 import moment from 'moment';
 import R from 'ramda';
 
-function createChangelog(newVersionNumber, validLabels, mergedPullRequests) {
-    const groupedPullRequests = R.groupBy(R.prop('label'), mergedPullRequests);
-    const date = moment().locale('en').format('MMMM D, YYYY');
-    const title = `## ${newVersionNumber} (${date})`;
+const COLLAPSE_COUNT = 10;
 
-    let changelog = `${title}\n\n`;
+function formatPR(pr) {
+    return `-   ${pr.title} [#${pr.id}](https://github.com/storybooks/storybook/pull/${pr.id})\n`;
+}
 
-    Object.keys(groupedPullRequests).forEach(function (label) {
-        const pullRequests = groupedPullRequests[label];
+function formatCollapsed(heading, pullRequests, options) {
+    let formatted = `#### ${heading}\n\n`;
+    formatted += `<details>\n<summary>\n${pullRequests.length} PRs\n</summary>\n\n`;
+    pullRequests.forEach((pullRequest) => {
+        formatted += formatPR(pullRequest);
+    });
+    formatted += '\n</details>\n\n';
+    return formatted;
+}
 
-        changelog += `### ${validLabels[label]}\n\n`;
+function formatExpanded(heading, pullRequests, options) {
+    let formatted = `#### ${heading}\n\n`;
+    pullRequests.forEach((pullRequest) => {
+        formatted += formatPR(pullRequest);
+    });
+    formatted += '\n';
+    return formatted;
+}
 
-        pullRequests.forEach(function (pullRequest) {
-            changelog += `* ${pullRequest.title} (#${pullRequest.id})\n`;
-        });
+function createChangelog(newVersionNumber, formatOptions, mergedPullRequests) {
+    const { validLabels, skipLabels } = formatOptions;
+    const groupedPullRequests = R.groupBy(R.prop('label'), mergedPullRequests);
+    const date = moment().locale('en').format('YYYY-MMMM-DD');
```

Comment:
Looks like you also changed the date format. We could make this a configurable option, what do you think?